Own build `vercel dev` gives "Cannot find module 'dotenv'"

**Describe the bug**

I've forked the project and successfully forked my own copy to Vercel (it is live and working fine on https://github.com/MikeBeaton).

I have then set up a local dev env on my machine (in order to see if I could find out anything about and potentially offer a PR for Issue #1011 - **EDIT:** Which I have in fact [now done](https://github.com/anuraghazra/github-readme-stats/pull/1013), but I haven't fixed _this_ issue!)

Following (closely!) the instructions at https://github.com/anuraghazra/github-readme-stats/blob/master/CONTRIBUTING.md#local-development , I have got as far as `vercel dev` correctly opening and starting a locally hosted version on http://localhost:3000/ . 

However, when I navigate to http://localhost:3000/api?username=MikeBeaton then I see a "Bad Gateway" error in the browser (screenshot below), and at the same time the console output from `vercel dev` contains the error "Cannot find module 'dotenv'" (full error also below).

**Expected behavior**

No error and successful generation of the requested image, as happens fine on the version running directly on Vercel.

**Screenshots / Live demo link (paste the github-readme-stats link as markdown image)**

![image](https://user-images.githubusercontent.com/11946605/115883361-9be35e00-a445-11eb-967b-af3d6aa3304f.png)

```
Cannot find module 'dotenv'
Require stack:
- /mnt/c/OpenSource/github-readme-stats/api/index.js
- /mnt/c/Users/m.beaton/AppData/Roaming/npm/node_modules/vercel/node_modules/@vercel/node/dist/launcher.js
- /mnt/c/Users/m.beaton/AppData/Roaming/npm/node_modules/vercel/node_modules/@vercel/node/dist/dev-server.js
Did you forget to add it to "dependencies" in `package.json`?
Error! `node api/index.js` failed with exit code 1
```


**Additional context**

I have tried node installed 'normally' on Windows with no nvm (i.e. at command prompt `where node` shows `C:\Program Files\nodejs\node.exe`) and I have also tried node using `nvm` in WSL (i.e. at WSL Bash prompt `which node` shows `/home/mjsbeaton/.nvm/versions/node/v14.16.1/bin/node`). Both versions can run `github-readme-stats` fine, and both give exactly the same error. (Also, the same installs of node/npm are building and running webpack projects as normal.)

I am building a fork of `github-readme-stats` which is up to date with the current `master` branch and which builds and runs fine when hosted directly on Vercel - so I think the issue must be something to do with running on Windows?

--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -24,6 +24,7 @@ _(make sure you already have a [vercel](https://vercel.com/) account)_
 
 1. Install [Vercel CLI](https://vercel.com/download)
 1. Fork the repository and clone the code to your local machine
+1. Run `npm install` in the repository root
 1. Run the command "vercel" in the root and follow the steps there
 1. Create a `.env` file in the root of the directory
 1. In the .env file add a new variable named "PAT_1" with your [github Personal access token](https://docs.github.com/en/github/authenticating-to-github/creating-a-personal-access-token)

